src/mmrelay/plugin_loader.py

```
@@ -884,311 +884,543 @@ def _raise_install_error(pkg_name):
-                            logger.warning(
-                                "Could not checkout any branch, using current state"
-                            )
-                            return True
-        except (subprocess.CalledProcessError, FileNotFoundError) as e:
-            logger.error(f"Error updating repository {repo_name}: {e}")
+                        # If we get here, we couldn't checkout any branch
+                        logger.warning(
+                            "Could not checkout any branch, using current state"
+                        )
+                        return True
+    except (subprocess.CalledProcessError, FileNotFoundError) as e:
```

Ruff (0.14.3)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)